src/main/webapp/pipe.js

```diff
@@ -1,6 +1,15 @@
 function pipelineUtils() {
     var self = this;
     this.updatePipelines = function(divNames, errorDiv, view, fullscreen, page, component, showChanges, aggregatedChangesGroupingPattern, timeout, pipelineid, jsplumb) {
+
+        // Don't refresh pipelines if the tab/window is not active
+        if (document.hidden) {
```

Comment:
It has been supported for quite a while, see https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API

If it is `undefined` (old browsers) then this will never evaluate to `true` and things will just continue working the same way they did before.